B/bat/build_tarballs.jl

```diff
@@ -0,0 +1,45 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, BinaryBuilderBase
+
+name = "bat"
+version = v"13.0.0"
+
+# Collection of sources required to complete build
+#
+sources = [
+    GitSource(
+        "https://github.com/sharkdp/bat.git",
+        "25f4f96ea3afb6fe44552f3b38ed8b1540ffa1b3"),
+]
+
+# For now build only minimal-application because of linking problems
+# when enabling git support with libgit2/zlib and apple platforms
+script = raw"""
+cd $WORKSPACE/srcdir/bat
+cargo build --no-default-features --features minimal-application --locked --release
+install -Dvm 755 "target/${rust_target}/release/bat${exeext}" -t "${bindir}"
+install_license LICENSE-MIT LICENSE-APACHE NOTICE
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+# Rust toolchain for i686 Windows is unusable
+filter!(p -> !Sys.iswindows(p) || arch(p) != "i686", platforms)
+# Required Rust toolchain not available for RISC-V or FreeBSD/aarch64
```

Comment:
Is there a specific Rust toolchain this needs? It would be nice if you could leave that in a comment here.

Reply:
No I don't think so. Maybe the wording is bad. BinaryBuilder just tries to get the newest and fails I think. I don't know if older versions are available -- and I don't know how to instruct BinaryBuilder to grab them if they are. It's just based on which runners were able to obtain Cargo "out-of-the-box"

Reply:
I do not think we have any working Rust toolchains for riscv64 nor for aarch64-freebsd. Someone with knowledge about installing a cross-build Rust environment could probably fix that. The standard way of downloading a Rust toolchain is not working for these architectures, and I do not know how to build one from source.

See the file `0_RootFS/Rust/build_tarballs.jl`.